src/HUDL/HUDL.cpp

```diff
@@ -41,62 +36,49 @@ uint16_t HUDL::getObjectDictionarySize() const {
 }
 
 void HUDL::updateLCD() {
-    switch (currentHUDLScreen) {
-    case PAGE_1: {// Set a new scope for this case
-        if (!setHeaders) {
-            lcd.clearLCD();
-            headerForCorner(TOP_LEFT, "Battery");
-            headerForCorner(TOP_RIGHT, "HI Temp");
-            headerForCorner(BOTTOM_LEFT, "RPM");
-            headerForCorner(BOTTOM_RIGHT, "MC Stat");
-            setHeaders = true;
-        }
+    if (!setHeaders) {
+        lcd.clearLCD();
+        headerForCorner(TOP_LEFT, "Bat %");
+        headerForCorner(TOP_RIGHT, "Temp");
+        headerForCorner(BOTTOM_LEFT, "RPM");
+        headerForCorner(BOTTOM_RIGHT, "MC Stat");
+        setHeaders = true;
+    }
 
-        // Set the battery voltage
-        char voltage[9];
-        std::sprintf(voltage, "%hu.%hu v", totalVoltage / 10, totalVoltage % 10);
-        dataForCorner(TOP_LEFT, voltage);
-
-        // Set the highest temp
-        uint16_t highestTemp = thermTemps[0];
-        for (uint16_t temp : thermTemps) {
-            if (temp > highestTemp) {
-                highestTemp = temp;
-            }
-        }
-        char temp[9];
-        std::sprintf(temp, "%hu.%hu C", thermTemps[0] / 100, thermTemps[0] % 100);
-
-        dataForCorner(TOP_RIGHT, temp);
-
-        // Set the rpm
-        char rpm[8];
-        std::sprintf(rpm, "%lu", actualPosition);
-        dataForCorner(BOTTOM_LEFT, rpm);
-
-        // Set the status word
-        char status[8];
-
-        if (statusWord == 0x21) {
-            std::sprintf(status, "STOP");
-        } else if (statusWord == 0x27) {
-            std::sprintf(status, "GO");
-        } else {
-            std::sprintf(status, "UNKNOWN");
-            //            std::sprintf(errorString, "There was an error with the Motor Controller: Unknown Error %x", statusWord);
-            //            currentHUDLScreen = ERROR_PAGE;
-        }
+    // Set the battery voltage
+    char voltage[9];
+    std::sprintf(voltage, "%hu.%hu v", totalVoltage / 10, totalVoltage % 10);
+    dataForCorner(TOP_LEFT, voltage);
 
-        dataForCorner(BOTTOM_RIGHT, status);
-    } break;
-    case ERROR_PAGE: {// Set a new scope for this page
-        if (!setHeaders) {
-            lcd.clearLCD();
+    // Set the highest temp
+    uint16_t highestTemp = thermTemps[0];
+    for (uint16_t temp : thermTemps) {
+        if (temp > highestTemp) {
+            highestTemp = temp;
         }
-        writeError(errorString);
-        break;
     }
+
+    char temp[9];
+    std::sprintf(temp, "%hu.%hu C", highestTemp / 10, highestTemp % 10);
+    dataForCorner(TOP_RIGHT, temp);
+
+    // Set the rpm
+    char rpm[8];
+    std::sprintf(rpm, "%u", actualPosition);
```

Comment:
I've been getting complaints from integration about RPM rolling over to negatives when they spin the motor backwards. It's not really related to this PR, but can you change this here and change the data type of `actualPosition`?
```suggestion
    std::sprintf(rpm, "%d", actualPosition);
```